src/test/config.sol

```diff
@@ -110,16 +110,16 @@ contract Config {
         //
         // Values for all system configuration changes
         //
-        afterSpell.line_offset =           750 * MILLION;           // Offset between the global line against the sum of local lines
-        afterSpell.pot_dsr =               8_00;                    // In basis points
+        afterSpell.line_offset =           680 * MILLION;           // Offset between the global line against the sum of local lines
```

Comment:
Why we are changing offset here?

Reply:
Because the global debt ceiling was changed in this spell. 100M down for `CRVV1ETHSTETH-A`, 30M up for `RWA002-A`. Btw, the test actually would pass without this change, as it checks if it's anywhere between `line_offset` and `line_offset * 2`, but I've changed it to reflect overall decrease in the gap:
https://github.com/makerdao/spells-mainnet/blob/71685d7813a00e8ca364b63fea4796adadfa24d7/src/DssSpell.t.base.sol#L692-L693

Reply:
Clear, I think it would be good to have a line about gap in checklist

Reply:
Haven't been really following the concrete numbers, but I think the 30M shouldn't be part of the gap change, as that 30M increment in `RWA-2` has the corresponding of the global debt ceiling as well. However the -100M from `CRVV1ETHSTETH-A` is correct as the global debt ceiling was reduced in that number when the local debt ceiling was already 0. So the reference value IMO (and just following the logic at a high level point of view) should be 650.

Reply:
> So the reference value IMO (and just following the logic at a high level point of view) should be 650

Thaks for flagging this, we discussed it at GovOps yesterday and we will fix it in the PR for the upcoming spell